core/src/main/java/ai/timefold/solver/core/impl/phase/scope/AbstractPhaseScope.java

```diff
@@ -260,16 +260,16 @@ public int getNextStepIndex() {
         return getLastCompletedStepScope().getStepIndex() + 1;
     }
 
-    public boolean isReconfigurationTriggered() {
-        return this.reconfigurationTriggered;
+    public boolean isSolverStuck() {
+        return this.solverStuck;
     }
 
-    public void triggerReconfiguration() {
-        this.reconfigurationTriggered = true;
+    public void triggerSolverStuck() {
```

Comment:
Arguably, `setSolverStuck(boolean ...)` gets rid of two methods, and it is in line with how other fields on the class are modified.